code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -272,6 +274,60 @@ var/global/list/datum/stack_recipe/clown_recipes = list ( \
 	..()
 
 /*
+ * Titanium
+ */
+/obj/item/stack/sheet/mineral/titanium
+	name = "titanium"
+	icon_state = "sheet-titanium"
+	singular_name = "titanium sheet"
+	force = 5
+	throwforce = 5
+	w_class = 3
+	throw_speed = 1
+	throw_range = 3
+	origin_tech = "materials=4"
+	sheettype = "titanium"
+	materials = list(MAT_TITANIUM=MINERAL_MATERIAL_AMOUNT)
+
+var/global/list/datum/stack_recipe/titanium_recipes = list ( \
+	new/datum/stack_recipe("titanium tile", /obj/item/stack/tile/mineral/titanium, 1, 4, 20), \
+	)
+
+/obj/item/stack/sheet/mineral/titanium/New(var/loc, var/amount=null)
+	recipes = titanium_recipes
+	pixel_x = rand(0,4)-4
```

Comment:
I know this is how the other sheets do it, but this is (0,4)-4, which is anywhere from -4 to 0, which means they're weirdly biased towards the left, most things that randomise their coordinates to something like ```rand(-4,4)```, to eliminate all bias (and yes that means removing the extra ```-4```)